### ISSUE TITLE
RPC-API: saveKey

### ISSUE THREAD

**Issue by maxtaco** [keybase/go#67]
A method to submit a public or private key to keyring and optionally keybase.io:

```go
void saveKey(string key, string keyPassword, bool push)
```
cc: @gabriel

**Comment by maxtaco** [comment]
@gabriel, this is likely going to change as part of device provisioning. Are you talking about adding a pgp key?

**Comment by gabriel** [comment]
Yeah.. if they sign up and don't add or create a pgp key, we would need a way for them to add one later?

Currently you can select a pgp key but only from gpg and only during signup. 

Would be good to be able to select a pgp key from gpg, or to generate a new one, or to paste/add one manually (if you don't have gpg).

**Comment by maxtaco** [comment]
Ok, great, @patrickxb knows this code well...
